### ISSUE TITLE
Write guest-posts on popular Drupal, WP, and CMS related blogs

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#126]
Which places should this happen?
* todo

Who is going to do this?
* todo

Follow-up from https://github.com/backdrop-ops/backdropcms.org/issues/122

**Comment by sutibun** [comment]
I can write a post. I already have two potential titles in mind. 

> The fork in the road to Drupal 8

and

> Drupal users pushing back on proposition "D8"

No steal my titles.

Anyway, if you combine this with what I wrote https://github.com/backdrop-ops/backdropcms.org/issues/124#issuecomment-163820233, then we can do the following

1. Publish a guest post on a popular blog read by Drupal users
2. Pay FB to promote that guest post to even more Drupal users


But concerning this specific ticket, I think the first thing we need to ask is:

- Where do you go to read Drupal news and stay up to date? (Maybe we can create a specific issue for this poll)
- What are the popular blogs that cater to Drupal users? (Monkey doesn't really know. Use Drupal but never was that into the community.)

If every Bd user here can reply, we can get a sense (well, mostly me because I'm clueless) of where our current Bd users read Drupal stuff and thus where we guest post so we can find more like our Bd users.

Basically, first let's figure out where we can guest post. Then we can worry about the content.

Though will add: we shouldn't try to target popular WP or other CMS related blogs. Just Drupal-related.

**Comment by cellear** [comment]
Todo: list of non-Drupal podcasts, blogs etc.

and...

List of recent "modern" "competitor" CMSes (for example, Ghost, October, Bloomreach)

**Comment by BWPanda** [comment]
I follow this blog: https://itsfoss.com/ and they have an option for submitting ideas for review: https://itsfoss.com/get-featured-on-its-foss/

**Comment by jenlampton** [comment]
@BWPanda I've submitted Backdrop as an idea, thanks for the recommendation!
